CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+* The `role_revoke` function now takes a `%Role{}`, and a `%Group{}` instead of
+  names. It returns a `%Group{}` with the associated roles as an attribute. The
+  original `role_revoke` can still be used for users.
```

Comment:
Gotcha. Cool. For some of the things I was working on, we wanted to find by id. So I figured I'd move forward instead of spending the time _now_ supporting both id and name.